{"uuid": "4dc84581-ac83-4abe-97dc-45ccfded9f00", "children": ["c5d9c261-96e8-4fab-bd5d-d6009d7819d4"], "befores": [{"name": "browser_crbt", "status": "passed", "start": 1734445035484, "stop": 1734445037218}], "afters": [{"name": "browser_crbt::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: disconnected: not connected to DevTools\n  (failed to check if window was closed: disconnected: not connected to DevTools)\n  (Session info: chrome=131.0.6778.140)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7D4F86CC5+28821]\n\t(No symbol) [0x00007FF7D4EF3850]\n\t(No symbol) [0x00007FF7D4D9578A]\n\t(No symbol) [0x00007FF7D4D80866]\n\t(No symbol) [0x00007FF7D4D81D1F]\n\t(No symbol) [0x00007FF7D4D95DD3]\n\t(No symbol) [0x00007FF7D4D6EFCD]\n\t(No symbol) [0x00007FF7D4E1D699]\n\t(No symbol) [0x00007FF7D4E0F2C0]\n\t(No symbol) [0x00007FF7D4DDA778]\n\t(No symbol) [0x00007FF7D4DDB8E1]\n\tGetHandleVerifier [0x00007FF7D52BFCCD+3408029]\n\tGetHandleVerifier [0x00007FF7D52D743F+3504143]\n\tGetHandleVerifier [0x00007FF7D52CB61D+3455469]\n\tGetHandleVerifier [0x00007FF7D504BDCB+835995]\n\t(No symbol) [0x00007FF7D4EFEB6F]\n\t(No symbol) [0x00007FF7D4EFA824]\n\t(No symbol) [0x00007FF7D4EFA9BD]\n\t(No symbol) [0x00007FF7D4EEA1A9]\n\tBaseThreadInitThunk [0x00007FF8CE20259D+29]\n\tRtlUserThreadStart [0x00007FF8CFE6AF38+40]\n\n", "trace": "  File \"C:\\Users\\LenT14G2ITL\\PycharmProjects\\Pythonassessment\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\LenT14G2ITL\\PycharmProjects\\Pythonassessment\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\LenT14G2ITL\\PycharmProjects\\Pythonassessment\\Test_scripts\\conftest.py\", line 93, in browser_crbt\n    driver.close()\n    ~~~~~~~~~~~~^^\n  File \"C:\\Users\\LenT14G2ITL\\PycharmProjects\\Pythonassessment\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 495, in close\n    self.execute(Command.CLOSE)\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File \"C:\\Users\\LenT14G2ITL\\PycharmProjects\\Pythonassessment\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\LenT14G2ITL\\PycharmProjects\\Pythonassessment\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1734445043680, "stop": 1734445043692}], "start": 1734445035484, "stop": 1734445043692}